translations/ui-plugin-find-license/en_US.json

```diff
@@ -10,5 +10,7 @@
     "prop.startDate": "Start date",
     "prop.status": "Status",
     "prop.type": "Type",
-    "permission.search": "Find License Plugin: Search licenses"
+    "permission.search": "Find License Plugin: Search licenses",
+    "prop.tags": "Tags",
+    "term": "Term"
```

Comment:
Adding new translations to `en.json` should suffice. We need to edit the `en_US` file only when making changes to existing translations